xoto3/dynamodb/write_versioned/ddb_api.py

```diff
@@ -133,7 +136,12 @@ def boto3_transact_multiple_but_optimize_single(TransactItems: List[dict], **kwa
             if set(command) == {"Delete"}:
                 ddb_client.delete_item(**{**item_args, **kwargs})
                 return
-            # we don't (yet) support single writee optimization for things other than Put or Delete
+            if set(command) == {"ConditionCheck"}:
```

Comment:
How would we end up with a solitary ConditionCheck?

Reply:
a put_item where the value is identical to what's currently in the items fetched.